docs/sources/enterprise/_index.md

```diff
@@ -6,75 +6,73 @@ type = "docs"
 [menu.docs]
 name = "Grafana Enterprise"
 identifier = "enterprise"
-weight = 30
+weight = 100
 +++
 
 # Grafana Enterprise
 
-Grafana Enterprise is a commercial edition of Grafana that includes additional features not found in the open source
-version.
+Grafana Enterprise is a commercial edition of Grafana that includes additional features not found in the open source version.
 
-Building on everything you already know and love about Grafana, Grafana Enterprise adds enterprise data sources,
-advanced authentication options, more permission controls, 24x7x365 support, and training from the core Grafana team.
+Building on everything you already know and love about Grafana, Grafana Enterprise adds enterprise data sources, advanced authentication options, more permission controls, 24x7x365 support, and training from the core Grafana team.
 
 Grafana Enterprise includes all of the features found in the open source edition and more.
 
-___
+[Learn more about Grafana Enterprise.](https://grafana.com/enterprise)
 
-### Enhanced LDAP Integration
+## Enhanced security features
 
-With Grafana Enterprise you can set up synchronization between LDAP Groups and Teams. [Learn More]({{< relref "../auth/enhanced_ldap.md" >}}).
+Grafana Enterprise includes high-level security integration.
 
-### SAML Authentication
+### Data source permissions
 
-Enables your Grafana Enterprise users to authenticate with SAML. [Learn More]({{< relref "../auth/saml.md" >}}).
+[Data source permissions]({{< relref "datasource_permissions.md" >}}) allow you to restrict query access to only specific teams and users.
 
-### Team Sync
+### Enhanced LDAP integration
 
-Team Sync allows you to setup synchronization between teams in Grafana and teams in your auth provider so that your users automatically end up in the right team. [Learn More]({{< relref "../auth/team-sync.md" >}}).
+With Grafana Enterprise, you can set up synchronization between LDAP groups and Grafana teams. [Learn more about enhanced LDAP.]({{< relref "enhanced_ldap.md" >}})
 
-Supported auth providers:
+### SAML authentication
 
-* [LDAP]({{< relref "../auth/enhanced_ldap.md#ldap-group-synchronization-for-teams" >}})
-* [GitHub OAuth]({{< relref "../auth/github.md#team-sync-enterprise-only" >}})
-* [GitLab OAuth]({{< relref "../auth/gitlab.md#team-sync-enterprise-only" >}})
-* [Auth Proxy]({{< relref "../auth/auth-proxy.md#team-sync-enterprise-only">}})
+[SAML authentication]({{< relref "saml.md" >}}) enables your Grafana Enterprise users to authenticate with SAML.
 
-### White labeling
+## Reporting
 
-White labeling makes it possible to customize the logos and footer links of Grafana. [Learn More]({{< relref "white-labeling.md" >}}).
+[Reporting]({{< relref "reporting.md" >}}) allows you to take any dashboard, generate a PDF report, and set up a schedule to have it emailed to whoever you choose.
 
+## Team sync
```

Comment:
This could be categorized under the enhanced security functionality.

Reply:
Done